utils/zerovec/src/ule/tuplevar.rs

```diff
@@ -167,6 +167,62 @@ macro_rules! tuple_varule {
                 )
             }
         }
+
+        #[cfg(feature = "serde")]
+        impl<$($T: serde::Serialize),+> serde::Serialize for $name<$($T),+>
+        where
+            $($T: VarULE + ?Sized,)+
+            // This impl should be present on almost all VarULE types. if it isn't, that is a bug
+            $(for<'a> &'a $T: ZeroFrom<'a, $T>),+
```

Comment:
Nit: You aren't using the `ZeroFrom` bound in this impl any more as far as I can tell